src/js/va-letters/utils/helpers.js

```diff
@@ -33,3 +35,14 @@ export function apiRequest(url, optionalSettings = {}, success, error) {
     })
     .then(success, error);
 }
+
+export const letterContent = {
+  commissary: 'If you\'re a Veteran who is permanently and totally disabled, use the letter to access the commissary on your local base.',
+  proof_of_service: 'This card serves as proof of honorable service in the uniformed services and can replace a VA ID card.',
+  medicare_partd: 'You will need this letter as proof that you qualify for Medicare Part D prescription drug coverage.',
+  minimum_essential_coverage: 'This letter shows that you have Minimum Essential Coverage (MEC). MEC means that your health plan meets the requirements for health insurance under the Affordable Care Act (ACA). You may also need this letter when you change health insurance plans to show what days you were covered by the plan.',
+  service_verification: 'This letter shows your branch of service, date entered on active duty, and date discharged from active duty.',
+  civil_service: 'You will need this letter to prove you are a disabled Veteran who can get preference for civil service jobs.',
+  benefit_summary: 'This letter shows what benefits you\'re receiving from the VA, military service, and disability status. Below, you can choose if you want military service and disability status to be included.',
```

Comment:
agreed, it would be great to get a complete list of all possible letters from EVSS, if we don't have that already